js/main.js

```diff
@@ -2,13 +2,78 @@
 /* exported data */
 var $photoURL = document.querySelector('.photoURL');
 var $image = document.querySelector('.image');
+var $entryForm = document.querySelector('.entry-form');
+var $title = document.querySelector('.title');
+var $notes = document.querySelector('.notes');
+var $ul = document.querySelector('ul');
+var $noEntries = document.getElementById('noEntries');
+var $newButton = document.querySelector('.newButton');
+var $entries = document.querySelector('.entries');
+var $view = document.querySelectorAll('.view');
+
+function viewSwap() {
```

Comment:
You should make this receive a parameter of the name of the view to switch to. You should also be updating the value of `data.view` in this function, that is the best place to do it, since you are changing the view here.